controllers/numaresourcesscheduler_controller.go

```diff
@@ -55,7 +55,8 @@ import (
 )
 
 const (
-	leaderElectionResourceName = "numa-scheduler-leader"
+	leaderElectionResourceName      = "numa-scheduler-leader"
+	SystemNodeCriticalPriorityClass = "system-node-critical"
```

Comment:
please make it private and rename to convey the intent, not the value. Maybe `schedulerPriorityClassName`

Reply:
In addition, please record in the commit message why `system-node-critical` and not `system-cluster-critical` (I'm fine with the decision, but let's record the reason)

Reply:
updated, thanks for raising